source/configure/site-configuration-settings.rst

```diff
@@ -329,6 +329,26 @@ iOS app download link
 | String input. Default is ``https://about.mattermost.com/mattermost-ios-app/``.                                                                                                                                        |                                                                        |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------+
 
+.. config:setting:: exp-appcustomurlschemes
+  :displayname: App custom URL schemes (Customization)
+  :systemconsole: N/A
+  :configjson: .NativeAppSettings.AppCustomURLSchemes
+  :environment: MM_NATIVEAPPSETTINGS_APPCUSTOMURLSCHEMES
+  :description: Define valid custom URL schemes for redirect links provided by custom-built mobile Mattermost apps.
+
+App custom URL schemes
+~~~~~~~~~~~~~~~~~~~~~~
+
+This setting isn't available in the System Console and can only be set in ``config.json``.
+
+Define valid custom URL schemes for redirect links provided by custom-built mobile Mattermost apps. This ensures users are redirected to the custom-built mobile app and not Mattermost's mobile client.
+
+When configured, after OAuth or SAML user authentication is complete, custom URL schemes sent by mobile clients are validated to ensure they don't include default schemes such as ``http`` or ``https``. Mobile users are then redirected back to the mobile app using the custom scheme URL provided by the mobile client. We recommend that you update your mobile client values as well with valid custom URL schemes.
+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"NativeAppSettings.AppCustomURLSchemes"`` with an array of strings as input separated by spaces. For example: ``[mmauth  mmauthbeta]``.  |
```

Comment:
"how would you suggest we show how to set this? I don't think it's extremely intuitive to do a space separated list unless instructed" - @coltoneshaw - is this sufficient? Or would you like to see this made more explicit?